types/paper/index.d.ts

```diff
@@ -1353,9 +1353,11 @@ declare module paper {
 
         /**
          * Clones the item within the same project and places the copy above the item.
-         * @param insert [optional] - specifies whether the copy should be inserted into the DOM. When set to true, it is inserted above the original. default: true
+         * @param options [optional] - object with 2 parameters
```

Comment:
seems like you have a comment specifying the shape of the object here, `{ insert?: boolean, deep?: boolean }` so why not use it?